### ISSUE TITLE
508-defect-2 [ZOOM, MOBILE DESIGN]: Screen content must be responsive to the viewport size

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#17469]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Jennifer

## User Story or Problem Statement

As a user who uses magnification plugins, I expect screen content to adjust to a magnified screen sizing, so that I am able to understand and interact with screen content.

## Details

When the screen is zoomed to 200% or 400%, the page content is cut off. 400% isn't too bad, but due to the tabs, it's not usable with the search results.

## Acceptance Criteria

- [x] When the screen is zoomed, the screen content should responsively render and be usable.

## Environment

* Operating System: all
* Browser: all
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/find-locations/?address=Portland%2C%20Maine%2004103%2C%20United%20States&context=Portland%2C%20Maine%2004103%2C%20United%20States&facilityType=health&location=43.69%2C-70.30&page=1&serviceType&searchString=Portland%2C%20Maine%2004103%2C%20United%20States#content` in browser
2. Use developer tools to set the browser width to 1280 pixels wide
3. Zoom the browser to 200%, then 400%
4. Verify that the map UI is cut off and it isn't possible to scroll to use see the button and map

## WCAG or Vendor Guidance (optional)

* [WCAG 2.0 Level A - 1.4.4, F69: Failure of Success Criterion 1.4.4 when resizing visually rendered text up to 200 percent causes the text, image or controls to be clipped, truncated or obscured](https://www.w3.org/TR/WCAG20-TECHS/F69.html)

## Screenshots or Trace Logs

![Screen Shot 2020-12-14 at 5 35 27 PM](https://user-images.githubusercontent.com/57469/102145654-9b9eda80-3e35-11eb-801d-58f693eb965d.png)

![Screen Shot 2020-12-14 at 5 32 34 PM](https://user-images.githubusercontent.com/57469/102145656-9c377100-3e35-11eb-865c-5f42c835707b.png)

**Comment by mmiddaugh** [comment]
The fix for this is on staging and available for your review @jenstrickland

**Comment by jenstrickland** [comment]
Nicely done! Thank you, validated. 🎉
